### PR TITLE
Feature/marimba ur5 integration

### PR DESCRIPTION
**Pull request by berkgungor**

### CODE REVIEW

File: marimba_ur5_moveit_config/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+cmake_minimum_required(VERSION 3.1.3)
+project(marimbabot_ur5_moveit_config)
```
Comment (Juphex) [inline marimba_ur5_moveit_config/CMakeLists.txt:2]: rename the folder from "marimba_ur5_moveit_config" to "marimbabot_ur5_moveit_config" for consistency ?

Comment (Juphex) [reply marimba_ur5_moveit_config/CMakeLists.txt:2]: also I guess that it is best practice that project and folder name match

**Review by Juphex — CHANGES_REQUESTED** [review]
do you use all files/are all files necessary?

like warehouse.launch, demo.launch

File: marimbabot_description/package.xml [hunk 2]
```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0"?>
 <package format="2">
-  <name>marimbabot_sim</name>
+  <name>marimbabot_description</name>
   <version>0.0.0</version>
-  <description>The marimbabot_sim package</description>
+  <description>The marimbabot_description package</description>
```
Comment (Flova) [inline marimbabot_description/package.xml:5]: Add more detailed description

**Review by Juphex — COMMENTED** [review]
except that I disagree with the boilerplate moveit gives, the rest
lgtm when the requested changes by the others are applied

File: marimbabot_ur5_moveit_config/.setup_assistant [hunk 3]
```diff
@@ -0,0 +1,11 @@
+moveit_setup_assistant_config:
+  URDF:
+    package: marimbabot_description
+    relative_path: urdf/marimbabot_ur5.urdf.xacro
+    xacro_args: ""
```
Comment (berkgungor) [inline marimbabot_ur5_moveit_config/.setup_assistant:5]: we can remove xacro_args since it's empty stirng

Comment (Flova) [reply marimbabot_ur5_moveit_config/.setup_assistant:5]: I think it is a nessesary argument even if it is empty

**Review by Flova — APPROVED** [review]

**Pull request merged**